patching_tool_version.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 source ./_common.sh
+source ./_liferay_common.sh
```

Comment:
Hey @lucasmiranda0, can you please adjust the commit message to be a little more clearer?

I mean, something like: `Get latest Patching Tool version from releases.liferay.com`